dangerfile.js

```diff
@@ -126,3 +126,42 @@ if (packageChanged && !lockfileChanged) {
   const idea = "Perhaps you need to run `yarn install`?";
   warn(`${message} - <i>${idea}</i>`);
 }
+
+/* CHECK BUNDLE FILESIZES */
+const warnSize = 150000; // 150kb
+const failSize = 200000; // 200kb
+const buildDir = "./web/dist/client/";
+
+if (fs.existsSync(buildDir)) {
+  // get bundle names
+  const files = fs.readdirSync(buildDir).filter(name => name.match(/.js$/));
+  const reducer = size => (over, filename) =>
+    fs.statSync(`${buildDir}${filename}`).size > size
+      ? over.concat({
+          filename,
+          size: fs.statSync(`${buildDir}${filename}`).size,
+        })
+      : over;
+  // const fails = files.reduce(reducer(failSize), []).map(x => `${x.filename}:${x.size/1000}kb`);
+  const over = files.reduce(reducer(warnSize), []);
+  const warns = over
+    .filter(x => x.size < failSize)
+    .map(x => `${x.filename}:${x.size / 1000}kb`);
+  const fails = over
+    .filter(x => x.size >= failSize)
+    .map(x => `${x.filename}:${x.size / 1000}kb`);
+  if (warns.length) {
+    warn(
+      `Filesizes are getting large (over ${warnSize / 1000}kb): ${warns.join(", ")}`,
```

Comment:
@JakeDawkins for the links you can use https://github.com/NewSpring/Rock-Native/blob/1aefbed9bd96c76a0aa7b3a7fdb0ed5b65fc62b3/dangerfile.js#L8-L15